src/scrapers/platformen/nijmegenoost.py

```diff
@@ -1,49 +1,18 @@
-import requests
-from bs4 import BeautifulSoup
+import logging
 
-from models import InitiativeImport, Db
+from platformen import Scraper
+from platformen.mijnbuurtje import MijnBuurtjeSource, MijnBuurtjeSourceConfig
```

Comment:
Misschien is het beter om een mijnbuurtje mapje te maken, deze mixed package imports zien er nu een beetje gek uit